roofit/roofit/src/RooGaussian.cxx

```diff
@@ -60,13 +60,9 @@ RooGaussian::RooGaussian(const RooGaussian& other, const char* name) :
 
 Double_t RooGaussian::evaluate() const
 {
-  Double_t arg= x - mean;
-  Double_t sig = sigma ;
-  Double_t ret =exp(-0.5*arg*arg/(sig*sig)) ;
-//   if (gDebug>2) {
-//     cout << "gauss(" << GetName() << ") x = " << x << " mean = " << mean << " sigma = " << sigma << " ret = " << ret << endl ;
-//   }
-  return ret ;
+  const double arg = x - mean;
+  const double sig = sigma;
+  return exp(-0.5*arg*arg/(sig*sig)) ;
```

Comment:
`return exp(-0.5*arg*arg/(sigma*sigma))` is too long? :-)

Reply:
No, it's not. However, sigma is a class, not a simple type. It has operator Double_t(), but no multiplication operator. Presumably, it's been implemented like this to avoid calling the conversion twice.
I don't know, honestly, if the optimiser is smart enough to figure this out directly.